/*
 * Обработка ошибок, "try..catch"
 */
try {
  // "опасный" код
  // выполнение кода прерывается ошибкой
} catch (error) {
  // обработка ошибок
  // если их нет, этот блок пропускается
} finally {
  // код, который всегда выполняется после блоков try или catch
}
//  error - объект ошибки
